### PR TITLE
Add a new option to bear and add the vomit story

### PR DESCRIPTION
**Pull request by jmmartinezdev**

### CODE REVIEW

**Comment by jmmartinezdev** [comment]
Please review this, @casey-collab

File: english/bear/bear.md [hunk 1]
```diff
@@ -2,4 +2,8 @@ Stella atroir.
 Taiwan bear.
 Ice cold. If warm, the results could be unbearable.
 Bear in mind, the grizzly taste.
-You are now drunk. You stumble over to the nearest cutie and ask, "Hey baby, what's ursine?"
\ No newline at end of file
+You are now drunk. You stumble over to the nearest cutie and ask, "Hey baby, what's ursine?"
+
+When she slaps you in the face...
+
+[You vomit on top of her](english/vomit/vomit.md)
```
Comment (casey-collab) [inline english/bear/bear.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/vomit/vomit.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Your dreams of vomiting on top of someone have been finally accomplished. Let's get cray!
```
Comment (casey-collab) [inline english/vomit/vomit.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jmmartinezdev** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**